### PR TITLE
VACMS 17299 Lovell TRICARE MHS Genesis portal link and VA Make an appointment link testing

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Adds testing now that EHR system is cerner on all platforms. Tests that MHS Genesis link is present on all necessary pages and Make an appointment link on VA pages. Tests that a user shouldn't be able to travel from VA Make an appointment page to TRICARE counterpart.
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17107

## Testing done

- Added cypress E2E testing for Lovell pages/links on TRICARE and VA

## Screenshots

No visual changes. Just testing.

## What areas of the site does it impact?

Does not impact the site.

## Acceptance criteria

- [x] Testing for Lovell TRICARE MHS Genesis links working for all Lovell TRICARE pages.
- [x] Testing for absence of Experience switcher on Lovell VA Make an appointment page

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated (N/A)
- [x] Screenshot of the developed feature is added (N/A)
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed (N/A)

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user (N/A)

## Requested Feedback

Check that tests pass in CI

### CODE REVIEW

File: src/site/tests/cypress/vacms-lovell.cypress.spec.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  it('TRICARE Health services has MHS Genesis portal link', () => {
+    cy.visit('/lovell-federal-health-care-tricare/health-services/');
+    cy.injectAxeThenAxeCheck();
+    cy.findByText('MHS Genesis Patient Portal').then($link => {
+      expect($link).to.exist;
+      const attr = $link.attr('href');
+      expect(attr).to.equal('https://my.mhsgenesis.health.mil/');
+    });
+  });
+  it('TRICARE Health services has MHS Genesis portal link', () => {
```
Comment (maxx1128) [inline src/site/tests/cypress/vacms-lovell.cypress.spec.js:22]: **Todo:** This is a duplicate of the above test and should be deleted.

**Review by maxx1128 — CHANGES_REQUESTED** [review]
One small test to remove then it's good.

File: src/site/tests/cypress/vacms-lovell.cypress.spec.js (hunk 1 above)
Comment (eselkin) [reply src/site/tests/cypress/vacms-lovell.cypress.spec.js:22]: Yep. Will do.

Comment (eselkin) [reply src/site/tests/cypress/vacms-lovell.cypress.spec.js:22]: Removed.

**Review by maxx1128 — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**